### DISCUSSION TITLE
How to mirror budibase user id and email (couchdb), to a user table in postgres?

### DISCUSSION THREAD

**Discussion by dnk8n** [Budibase/budibase#10066] [Help]
I wanted to create an automation that was triggered when creating a new user, but I can't see an option for such a trigger.

My Use Case, is that I have other systems that read from the same postgres database, but I am missing employee information, which I want to relate to Budibase Users... so I need some way of linking the budibase user to the rest of the postgres data.

example of `employee` table, would have the following columns:
 - budibase_user_id: ro_ta_users_us_***
 - email: ***@***.com
 
 In the other system (metabase) I need to be able to perform sql queries to filter by employee.email (which would match with User.email)

**Comment by melohagan** [comment]
Hey @dnk8n 

The users table is a special table that doesn't support the **Row Created** trigger because users are created and handled outside of the Budibase app, through User Management in the portal.

To join data from different datasources, you could have a look at this tutorial: https://docs.budibase.com/docs/js-scripting#tutorial-linking-an-api-response-to-table-rows

The general idea would be to do a **Query Rows** action for the *Users* internal Budibase table, as well as the other source. Then you can use JS Scripting to "merge" the data, which could then be used to save to another table, or update an existing row.

![Screenshot 2023-03-21 at 15 37 40](https://user-images.githubusercontent.com/101575380/226658857-ec55cdfb-5a46-4ee2-889a-56215b39ccd5.png)
